BirthdayGreetingsTest/bootstrap.js

```diff
@@ -0,0 +1,14 @@
+import { Friend } from "./src/models/Friend.js";
+
+export const addData = async () => {
+  try {
+    Friend.create({
```

Comment:
```suggestion
    await Friend.create({
```
`If you're using an Async function do you expect this to be awaited?

Reply:
@RichardWatkins1 Missed this, it was working without it being awaited so I'm guessing it doesn't need to be async?